examples/read_log.py

```diff
@@ -0,0 +1,53 @@
+from pyspecdata import *
+from pyspecProcScripts import logobj
+import h5py
+from matplotlib.pyplot import subplots
+from numpy import log10
+
+coupler_atten = 22
+myfilename = search_filename(
+    "230626_batch230515_E37_Ras_B10_ODNP_1.h5",
+    exp_type="ODNP_NMR_comp/ODNP",
+    unique=True,
+)
+with figlist_var() as fl:
+    # {{{ open h5 file to real log
+    with h5py.File(myfilename, "r") as f:
+        log_grp = f["log"]
+        thislog = logobj.from_group(log_grp)
+        read_array = thislog.total_log
+        read_dict = thislog.log_dict
+    # }}}
+    # {{{ separate the time, RX, power, and commands from the
+    #     log array
+    for j in range(len(read_array)):
+        thistime, thisrx, thispower, thiscmd = read_array[j]
+    # }}}
+    # In order to properly set the time axis to start at 0
+    # both the log's start time will be subtracted from the
+    # the relative time recorded
+    log_start_time = read_array["time"][0]
+    relative_time = read_array["time"]
+    time_axis = relative_time - log_start_time
```

Comment:
Again, these three lines can be one line... you only want time_axis

Reply:
I thought it would be more legible for someone reading this but if you think otherwise-I changed it